soccer/Robot.cpp

```diff
@@ -235,6 +236,10 @@ const Geometry2d::Segment OurRobot::kickerBar() const {
     return Segment(pose * L, pose * R);
 }
 
+Geometry2d::Point OurRobot::mouthPos() const {
```

Comment:
Just making sure, this is just a utility method right (you're not using it anywhere here). If so, can you add it to the python bridge too (that'd be useful).